### ISSUE TITLE
Book 05: Sentence 026

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#50]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/a0b00212-fe01-4d65-9eb5-1c7e17981f84)

```
# sent_id = Book_05_-_The_Omens.txt:26
# text = Ic ei capitulo, vncan moteneoa: in tetzaujtl catca, in tetzammachoia: in jquac iooltica qujcaquja, in juhquj aca, tlatepuzmjuja: in juhquj quauhxeloaia, in qujtoaia, ioaltepuztli.
# text[norm] = Ic eyi capítulo, oncan motenehua: in tezahuitl catca, in tetzamachoya: in ihcuac yohualtica quicaquiya, in iuhqui aca, tlatepozmihuia: in iuhqui cuauhxelohuaya, in quihtoaya, yohualtepoztli.
# text[orig] = Ic·ei·capitulo·,·vncan·motene¶oa·:·intetzaujtl·catca·,·intetzam¶machoia·:·injquac·iooltica·quj¶caquja·,·injuhquj·aca·,·tlatepuzmj¶uja·:·injuhquj·quauhxeloaia·,·in¶qujtoaia·,·ioaltepuztli·.
# tagged = 97.06%
# analysed = 94.12%
1	Ic	ic	ADV	_	_	3	amod	_	_
2	ei	eyi	NUM	_	_	1	_	_	_
3	capitulo	capítulo	NOUN	_	State=Abs	6	obl	_	_
4	,	,	PUNCT	_	_	6	punct	_	_
5	vncan	_	ADV	_	_	6	advmod	_	_
6	moteneoa	tenehua	VERB	_	_	0	root	_	_
7	:	:	PUNCT	_	_	10	punct	_	_
8	in	in	SCONJ	_	_	9	_	_	_
9	tetzaujtl	tezahuitl	NOUN	_	State=Abs	10	_	_	_
10	catca	_	VERB	_	Subcat=Intr	6	_	_	_
11	,	,	PUNCT	_	_	13	punct	_	_
12	in	in	SCONJ	_	_	13	_	_	_
13	tetzammachoia	_	X	_	_	9	_	_	_
14	:	:	PUNCT	_	_	18	punct	_	_
15	in	in	SCONJ	_	_	18	_	_	_
16	jquac	ihcuac	ADV	_	_	18	_	_	_
17	iooltica	yohualtica	ADV	_	_	18	_	_	_
18	qujcaquja	caqui	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin13	_	_	_	_
19	,	,	PUNCT	_	_	24	punct	_	_
20	in	in	SCONJ	_	_	24	_	_	_
21	juhquj	iuhqui	ADV	_	_	24	_	_	_
22	aca	_	PRON	_	_	24	_	_	_
23	,	,	PUNCT	_	_	24	punct	_	_
24	tlatepuzmjuja	tepozmihuia	VERB	_	_	18	_	_	_
25	:	:	PUNCT	_	_	28	punct	_	_
26	in	in	SCONJ	_	_	28	_	_	_
27	juhquj	iuhqui	ADV	_	_	28	_	_	_
28	quauhxeloaia	_	VERB	_	_	24	_	_	_
29	,	,	PUNCT	_	_	31	punct	_	_
30	in	in	SCONJ	_	_	31	_	_	_
31	qujtoaia	ihtoa	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin13	_	_	_	_
32	,	,	PUNCT	_	_	33	punct	_	_
33	ioaltepuztli	yohualtepoztli	NOUN	_	State=Abs	31	_	_	_
34	.	.	PUNCT	_	_	6	punct	_	_
```

**Comment by ftyers** [comment]
López Austin:
> Capítulo tercero, donde se dice del agüero, de lo que era conocido como augurio cuando en la noche alguno oía como que golpeaban con un objeto metálico, como si rajaran maderos: decían que era el hacha nocturna.

**Comment by ftyers** [comment]
Update:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/c2ed0667-35f6-418c-ac62-734b182a5f22)

Questions:
- What do do with *iuhqui*, note no *-n* here...
- Is *yohualtepoztli* the `obj` of *quihtoaya*?
- This nested `csubj` is what we've done in other cases like this, is it right?

**Comment by Lguyogiro** [comment]
> What do do with iuhqui, note no -n here...

I think the treatment above is okay. It is like #7

**Comment by Lguyogiro** [comment]
> Is yohualtepoztli the obj of quihtoaya?
> This nested csubj is what we've done in other cases like this, is it right?

I think yes to both. I admit something about the `csubj` I don't love...but at the same time I've tried alternative analyses (e.g. what if those are all adverbial clauses of a main nominal clause _yohualtepoztli_ "it is the night axe") but I don't think they work given the distribution of _in_s.

**Comment by ftyers** [comment]
Like this?

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/8a1bbb1f-a7bc-4359-9b0c-6a880a4bc91b)

**Comment by Lguyogiro** [comment]
I think the rel from 10 -> 24 is `ccomp` instead of `csubj` here, since 24 is the head of the clause that describes what was heard?

Other than that, LGTM

**Comment by ftyers** [comment]
Hmm, 10 is *catca* ... did you mean 18 ? If so, then merged in 416b959.
